### DISCUSSION TITLE
Could not delete temporary directory when batect cleaning up

### DISCUSSION THREAD

**Discussion by jgv115** [batect/batect#973] [Q&A] [answered]
I was wondering if anyone had any idea on how to solve this issue.

In some batect tasks I'm running, when the container is cleaning up I get:

```
Error: Could not delete temporary directory '/tmp/batect-home-11561350591424246435'.
java.nio.file.AccessDeniedException: /tmp/batect-home-11561350591424246435/.nuget
```

I'm not sure where to start to solve this issue so any help would be appreciated

**Comment by charleskorn** [comment]
Hi @jgv115, are you using the most recent version of Batect? [v0.72](https://github.com/batect/batect/releases/tag/0.72.0) removed the use of temporary files, so errors like this should no longer happen.

**Comment by jgv115** [comment, marked as answer]
Thanks @charleskorn, i upgraded and it solved the issue!
